canonical-web-and-design/maas-ui
# Filter out duplicate alias members from interface list.

## Done
- Filter out duplicate alias members from interface list.

## QA
- Go to the networking page of a New, Allocated or Ready machine.
- Add a physical interface.
- Create a bridge from that interface.
- Add an alias of the bridge.
- Check that the physical interface only shows up once in the list, and that the alias does not show

## Fixes
Fixes #1219 

## Launchpad Issue
[lp#1881804](https://bugs.launchpad.net/maas/+bug/1881804)
